apps/calendar/js/views/pan.js

```diff
@@ -188,8 +188,9 @@ Pan.prototype = {
 
   _set: function(x) {
     x = clamp(x, this._minX, 0);
+    var _x = (document.documentElement.dir === 'rtl') ? -x : x;
```

Comment:
maybe we will really need to change the panning for RTL, it feels weird the way it is right now.